generators/client/files-angular.js

```diff
@@ -409,7 +417,7 @@ const files = {
             ]
         },
         {
-            condition: generator => !generator.skipUserManagement,
+            condition: generator => !generator.skipUserManagement || generator.authenticationType === 'oauth2',
```

Comment:
No, but there's pieces of user management that do exist because I sync the user from the IdP with the user in JHipster upon login. The reason for this is so developers can create relationships with the `User` domain object.